include/rosdiscover-clang/BackwardSymbolizer/FunctionSymbolizer.h

```diff
@@ -687,8 +690,117 @@ class FunctionSymbolizer {
     return expr->getConstructor()->getCanonicalDecl();
   }
 
+  // Recursively builds a graph of control dependencies starting from the last block.
+  std::vector<CFGBlock*> buildGraph(bool first, const clang::CFGBlock* block, const llvm::SmallVector<clang::CFGBlock *, 4> &deps, clang::CFGDominatorTreeImpl<true> &postdominatorAnalysis, clang::CFGDominatorTreeImpl<false> &dominatorAnalysis, std::vector<const clang::CFGBlock*> &analyzed, std::vector<CFGBlock*> &controlDependencyGraphNodes, std::unordered_map<long, CFGBlock*> &blockMap) {
```

Comment:
for better readability, it would be nice to split these separate function arguments across multiple lines

Reply:
fixed